### PR TITLE
Book Detail IV - Badge

### PR DESCRIPTION
**Pull request by joacomiguens**
## Summary

- Cover badge added

## Screenshot

image

## Trello Card

You can find it [here](https://trello.com/c/JrMNoUkT/29-detalle-de-libro-iv-badge)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <div class="detail">
       <div class="detail__cover">
         <img src="./assets/book-cover.png" alt="cover of the book" class="detail__cover__img" />
+        <img src="./assets/badge.png" alt="badge of the book" class="detail__cover__badge" />
```
Comment (ebedoyaalzate) [inline index.html]: Remember to use as less html code as possible for clarity and reusability. In this case, try just to use just the container element and apply the image as a decorator using only CSS (hint: you can use the ::before selector), that way you will have a single reusable element (the container) that contains the main image, and manage the decorative part via CSS only. If you have any doubt about this ask me.

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (joacomiguens) [reply index.html]: Done 😄

File: scss/_detail.scss [hunk 2]
```diff
@@ -14,8 +14,17 @@
   display: flex;
   max-width: 650px;
   padding: 16px;
+  position: relative;
   width: 100%;
 
+  &::before {
+    bottom: 50px;
+    content: url('./assets/badge.png');
```
Comment (ebedoyaalzate) [inline scss/_detail.scss]: You should use` background-image ` and `content: ""` attribute

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: scss/_detail.scss (hunk 2 above)
Comment (joacomiguens) [reply scss/_detail.scss]: Done 😄

**Review by ebedoyaalzate — APPROVED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**